recipes/clustalo/build.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
 
+./configure --prefix=$SRC_DIR
+make
+make install
```

Comment:
I think this shouldn't be necessary given that the binary is copied by hand later on?